File: Python3/258. Add Digits.py
class Solution:
    def addDigits(self, num):
        """
        :type num: int
        :rtype: int
        """
        # 关键在于Go中 -1%9 = -1 而python3中 -1%9 = -8
        return (num-1)%9 + 1 if num > 0 else 0 # 64ms, 其他的方法没什么意思

File: Golang/258. Add Digits.go
func addDigits(num int) int { // 8ms
    // if num == 0 {
    //     return 0
    // } else {
    //     return (num-1)%9 + 1
    // }

    return (num-1)%9 + 1 // 4ms // 关键在于Go中 -1%9 = -1 而python3中 -1%9 = -8
}
